File: runtime/ruby/units/rt/f32_copysign.rb
# requires: rt/f32_bits, rt/f32_from_bits
def f32_copysign(a, b)
  f32_from_bits((f32_bits(a) & 0x7fff_ffff) | (f32_bits(b) & 0x8000_0000))
end

File: runtime/perl/units/rt/f32_copysign.pl
# requires: rt/f32_bits, rt/f32_from_bits
sub f32_copysign {
    return Rt::f32_from_bits((Rt::f32_bits($_[0]) & 0x7FFFFFFF) | (Rt::f32_bits($_[1]) & 0x80000000));
}
